api/src/main/java/com/joellensilva/gqso/App.java

```diff
@@ -7,6 +7,7 @@ public class App extends Jooby {
   {
 
     get("/", ctx -> "Welcome to Jooby!");
```

Comment:
Mude a frase na raiz, por favor.
```suggestion
    get("/", ctx -> "Bem-vindo à nossa API!");
```